ui/components/app/confirmations/alerts/multiple-alert-modal/multiple-alert-modal.tsx

```diff
@@ -145,6 +181,7 @@ export function MultipleAlertModal({
   ownerId,
 }: MultipleAlertModalProps) {
   const { alerts } = useAlerts(ownerId);
+  const processAction = useConfirmationAlertActions();
```

Comment:
We still don't want to couple the confirmation alert handlers to this component as the `MultipleAlertModal` is also decoupled from confirmations given the `ownerId` property.

Should we put it in `ui/pages/confirmations/confirm/confirm.tsx` in the main confirmation template as we're coupled to a confirmation at that point?

Reply:
Done.